polars/polars-lazy/polars-plan/src/dsl/string.rs

```diff
@@ -456,14 +456,26 @@ impl StringNameSpace {
     }
 
     #[cfg(feature = "string_from_radix")]
-    /// Parse string in base radix into decimal
+    /// Parse string in base radix into decimal.
     pub fn from_radix(self, radix: u32, strict: bool) -> Expr {
         self.0
             .map_private(FunctionExpr::StringExpr(StringFunction::FromRadix(
                 radix, strict,
             )))
     }
 
+    /// Return the number of characters in the string (not bytes).
+    pub fn n_chars(self) -> Expr {
```

Comment:
Nice, can we now properly dispatch the python side to those?

Reply:
Oops! Missed the final link in the chain 😅 Should be good now - jumping on a plane in about an hour (at the airport now), so if there's anything else then I'll pick it up when I land and get home this evening. (In the meantime, let's see if I can get anything done from the emergency exit row at 10km up :)

Reply:
All good! You will land in a world with this PR merged. ;)